aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -45,6 +45,13 @@ namespace at {
 namespace vec256 {
 // See Note [Acceptable use of anonymous namespace in header]
 namespace {
+// at::Half should be treated as floating point
+template <typename T>
+struct is_floating_point:
+    std::integral_constant<bool,
+      std::is_floating_point<T>::value ||
+      std::is_same<T, at::Half>::value> {
+};
```

Comment:
Hmmmm... are you sure we don't already have a concept for this elsewhere in the codebase?

Reply:
Not as far as I could see. grepping for `is_floating_point` didn't show anything and I had a look in `Half.h`